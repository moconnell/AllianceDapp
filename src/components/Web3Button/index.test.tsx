import { render } from "@testing-library/react";
import Web3Button from "./index";

describe("<Web3Button />", () => {
  it("should render correctly", async () => {
    const { asFragment } = render(<Web3Button />);
    expect(asFragment()).toMatchSnapshot();
  });
});
