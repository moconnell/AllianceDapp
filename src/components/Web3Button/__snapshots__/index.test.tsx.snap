// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Web3Button /> should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ fqrVXb"
  >
    <button
      class="sc-gsnTZi hFXnKB"
    >
      Loading...
    </button>
  </div>
</DocumentFragment>
`;
